### DISCUSSION TITLE
Docker compose example

### DISCUSSION THREAD

**Discussion by arlgnv** [CardanoSolutions/kupo#133] [Q&A] [answered]
Hi. Do you have docker-compose example file which uses cardano-node-ogmios and kupo services together?

**Comment by KtorZ** [comment]
Hey, not quite yet. That's been on my TODO-list for a while for never sat down to do it. There's one compose example in Ogmios which wouldn't be too hard to extend with Kupo: https://github.com/CardanoSolutions/ogmios/blob/master/docker-compose.yml

[a comment by an automated account is not shown]
